tsl/src/data_node.c

```diff
@@ -2056,14 +2056,16 @@ Datum
 data_node_ping(PG_FUNCTION_ARGS)
 {
 	const char *node_name = PG_ARGISNULL(0) ? NULL : PG_GETARG_CSTRING(0);
+	TimestampTz timeout = PG_ARGISNULL(1) ? TS_NO_TIMEOUT : PG_GETARG_TIMESTAMPTZ(1);
```

Comment:
This has to be `PG_GETARG_INTERVAL_P` right?

Reply:
and type has to be `Interval`

Reply:
Changed type to timestamptz in the function definition